### PR TITLE
Show attendee role in free busy modal

### PR DESCRIPTION
**Pull request by st3iny**
Ref #39

Screenshot 2021-06-04 at 10-28-52 June 2021 - Calendar - Nextcloud

### CODE REVIEW

File: src/components/Editor/FreeBusy/FreeBusy.vue [hunk 1]
```diff
@@ -169,9 +169,14 @@ export default {
 			const resources = []
 
 			for (const attendee of [this.organizer, ...this.attendees]) {
+				let title = attendee.commonName || attendee.uri.substr(7)
+				if (attendee === this.organizer) {
+					title += ` ${this.$t('calendar', '(organizer)')}`
```
Comment (tcitworld) [inline src/components/Editor/FreeBusy/FreeBusy.vue:174]:
```suggestion
					title = ` ${this.$t('calendar', '{organizer} (organizer)', {organizer: title})}`
```
For correct i18n for languages in which the order of words might not be the same.

**Review by tcitworld — CHANGES_REQUESTED** [review]

File: src/components/Editor/FreeBusy/FreeBusy.vue (hunk 1 above)
Comment (st3iny) [reply src/components/Editor/FreeBusy/FreeBusy.vue]: Done

File: src/components/Editor/FreeBusy/FreeBusy.vue [hunk 2]
```diff
@@ -169,9 +169,16 @@ export default {
 			const resources = []
 
 			for (const attendee of [this.organizer, ...this.attendees]) {
+				let title = attendee.commonName || attendee.uri.substr(7)
+				if (attendee === this.organizer) {
+					title = this.$t('calendar', '{organizer} (organizer)', {
+						organizer: title
```
Comment (tcitworld) [inline src/components/Editor/FreeBusy/FreeBusy.vue:175]: Lint issue

**Review by ChristophWurst — APPROVED** [review]
Very nice

**Review by tcitworld — APPROVED** [review]

File: src/components/Editor/FreeBusy/FreeBusy.vue (hunk 2 above)
Comment (ChristophWurst) [reply src/components/Editor/FreeBusy/FreeBusy.vue:175]: oh no :see_no_evil:

Comment (ChristophWurst) [reply src/components/Editor/FreeBusy/FreeBusy.vue:175]: but CI is happy?

**Pull request merged**